ta/pkcs11/src/processing_symm.c

```diff
@@ -570,6 +701,10 @@ enum pkcs11_rc step_symm_operation(struct pkcs11_session *session,
 			TEE_Panic(function);
 			break;
 		}
+
+		if (rc == PKCS11_CKR_OK && proc->extra_ctx)
+			TEE_Free(proc->extra_ctx);
```

Comment:
I think this should be called from `release_active_processing()`.

Reply:
Yes you are right. I have shifted it there